mkosi/state.py

```diff
@@ -21,6 +21,16 @@ def __init__(self, args: MkosiArgs, config: MkosiConfig, workspace: Path) -> Non
         self.pkgmngr.mkdir()
         self.install_dir.mkdir(exist_ok=True)
         self.cache_dir.mkdir(parents=True, exist_ok=True)
+        source_date_epoch = self.config.environment.get("SOURCE_DATE_EPOCH")
+        self.source_date_epoch: Optional[int] = None
```

Comment:
It's nice to just set `self.source_date_epoch` at the end. It's not nice to set it to one value first and to a different one later.